mm/ktsan/func.c

```diff
@@ -1,5 +1,31 @@
 #include "ktsan.h"
 
+#include <linux/preempt.h>
+
+static void kt_tame_acquire_and_release_clocks(kt_thr_t *thr)
+{
+	if (in_interrupt())
+		return;
+
+	if (thr->acquire_active) {
+		thr->acquire_tame_counter++;
+		if (thr->acquire_tame_counter == KT_TAME_COUNTER_LIMIT) {
+			thr->acquire_active = false;
+			thr->acquire_tame_counter = 0;
+			kt_clk_init(&thr->acquire_clk);
```

Comment:
To get rid of the induced synchronization from the previous acquire.
Otherwise, on the next load-acquire we will also pick up clocks from the previous acquire.